sdk/agricultureplatform/Azure.ResourceManager.AgriculturePlatform/tsp_client_metadata.yaml

```diff
@@ -0,0 +1,24 @@
+version: 0.29.1
+dateCreatedOrModified: 2025-11-05T18:05:55.591Z
```

Comment:
This will force a regen even when you only changed something like test cases, we should rethink this file